File: my-app/src/types/category.ts
export interface Category {
  id: number;
  name: string;
  description?: string;
  slug?: string;
  isActive: boolean;
  sportsCount?: number;
  created?: string;
}

export interface CreateUpdateCategory {
  name: string;
  description?: string;
  slug?: string;
  isActive?: boolean;
}

File: API/Dms.Domain/Entities/Category.cs
using Dms.Domain.Common;

namespace Dms.Domain.Entities
{
    public class Category : BaseEntity
    {
        public string Name { get; set; } = string.Empty;
        public string? Description { get; set; }
        public string? Slug {  get; set; }
        public bool IsActive { get; set; } = true;
        public virtual ICollection<Sport> Sports { get; set; } = new List<Sport>();
    }
}
